flow/table.go

```diff
@@ -248,10 +250,14 @@ func (ft *Table) update(updateFrom, updateTime int64) {
 	logging.GetLogger().Debugf("flow table update: %d, %d", updateFrom, updateTime)
 
 	var updatedFlows []*Flow
-	for _, f := range ft.table {
+	for k, f := range ft.table {
 		if f.XXX_state.updateVersion > ft.updateVersion {
 			ft.updateMetric(f, updateFrom, updateTime)
+			f.ReportCounter++
 			updatedFlows = append(updatedFlows, f)
+			if f.FinishType != "" {
```

Comment:
that is an integer, I think the comparison if wrong

Reply:
oops :) fixed (I hope)